### ISSUE TITLE
Files missing when used in Mac OS

### ISSUE THREAD

**Issue by integral-llc** [linq2db/linq2db#1123]
I'm trying to install the T4 models under MacOS using .NET core, but I cannot find the `t4model` and the `Copy...` file as described in your readme file. 

Here is the output
```
Eugenius-MacBook-Pro:testDB dev$ dotnet add package linq2db.t4models
  Writing /var/folders/hw/lp2y92_x1c3_rx26pqkt15yw0000gn/T/tmpWnSYjB.tmp
info : Adding PackageReference for package 'linq2db.t4models' into project '/Users/dev/projects/learn/testDB/testDB.csproj'.
log  : Restoring packages for /Users/dev/projects/learn/testDB/testDB.csproj...
info :   CACHE https://api.nuget.org/v3-flatcontainer/linq2db.t4models/index.json
warn : Package 'linq2db 1.10.2' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.0'. This package may not be fully compatible with your project.
info : Package 'linq2db.t4models' is compatible with all the specified frameworks in project '/Users/dev/projects/learn/testDB/testDB.csproj'.
info : PackageReference for package 'linq2db.t4models' version '1.10.0' added to file '/Users/dev/projects/learn/testDB/testDB.csproj'.
```

If I list the files in the directory there is no folders `...Tools`
```
Eugenius-MacBook-Pro:testDB dev$ ls -l
total 16
-rw-r--r--  1 dev  staff  188 Apr 29 16:07 Program.cs
drwxr-xr-x  6 dev  staff  192 Apr 29 16:07 obj
-rw-r--r--  1 dev  staff  399 Apr 29 16:09 testDB.csproj
```

The `.csproj` file contains the correct references.

I also tried to do all this using `JetBrains Rider` app, but got same result. 

Please advise.

Thx!

### Environment details
linq2db version: *1.10.0*  
Operating system: *E.g. Mac OS High Sierra*  
Framework version: *.NET Core 2.1.203*

**Comment by MaceWindu** [comment]
For .net core and T4 I would recommend to switch to 2.0rc package from myget.

**Comment by ili** [comment]
Not sure if `linq2db.t4models` 1.xx.xx can be applied on non windows system and out of visual studio. 
Please, try the latest `linq2db.YourServer` from MyGet feed, we'v done some improvements for T4 packages, this can solve the problem (minimum version is  `2.0.0-rc2190`)

Looking forward for your feedback!

**Comment by MaceWindu** [comment]
We just released 2.0 beta 5 packages on nuget. It should handle remaining issues with core projects.
